lib/ts/recipe/thirdpartyemailpassword/utils.ts

```diff
@@ -42,6 +42,7 @@ export function normaliseThirdPartyEmailPasswordConfig(config: Config): Normalis
 
     const override: any = {
         functions: (originalImplementation: RecipeImplementation) => originalImplementation,
+        components: {},
         ...config.override,
     };
```

Comment:
(This comment is not related to this file, but related to `lib/ts/recipe/thirdpartyemailpassword/recipe.ts`) You also need to pass the component override to the emailpassword and thirdparty recipe instances inside the recipe clas of thirdpartyemailpassword